contracts/zero-ex/contracts/src/features/FundRecoveryFeature.sol

```diff
@@ -59,7 +59,7 @@ contract FundRecoveryFeature is
         if(amountOut == uint256(-1)) {
             amountOut = LibERC20Transformer.getTokenBalanceOf(erc20, address(this));
         }
-        LibERC20Transformer.transformerTransfer(erc20, recipientWallet, amountOut);
```

Comment:
IMO I would leave this unchanged. I'm actually scared that some features makes use of `LibERC20Transformer`, feels risky. 

I'd suggest keeping the old `transformerTransfer` and creating a duplicate `unsafeTransformerTransfer` to keep this audited feature unchanged.  

All the other Transformers are open season to modify though and make use of `unsafeTransformerTransfer`.

IMO in the future Features shouldn't touch `LibERC20Transformer` due to different security assumptions, but this is out of scope for this PR.

Reply:
Sounds good. -- Done!